austation/code/modules/awaymissions/corpse.dm

```diff
@@ -0,0 +1,11 @@
+/obj/effect/mob_spawn/human/bushranger
+	name = "Bushranger"
+	outfit = /datum/outfit/bushranger
+
+/datum/outfit/bushranger
+	name = "Bushranger"
+	uniform = /obj/item/clothing/under/color/grey
+	suit = /obj/item/clothing/suit/armor/nedarmour
+	head = /obj/item/clothing/head/helmet/nedhelmet
+	shoes = /obj/item/clothing/shoes/sandal
+	gloves = /obj/item/clothing/gloves/color/brown
```

Comment:
Not sure what that no newline business is all about

Reply:
git warns you when you don't have a newline at the end of your file